<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgBoxDialog</class>
 <widget class="QDialog" name="MsgBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MsgBoxDialog</string>
  </property>
  <widget class="QPushButton" name="pbButton2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Button2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbButton1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Button1</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teMessage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>440</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
